kmultiversx/src/kmultiversx/kdist/mx-semantics/esdt.md

```diff
@@ -148,11 +148,13 @@ module ESDT
           ...
         </account>
         <instrs> (#waitCommands ~> _) #Or .K </instrs>
-      [priority(61)]
+      [priority(61), preserves-definedness]
+      // preserves-definedness: ACCT exists prior so the account map is defined, and the case where
+      // TOKEN exists prior in esdtData is handled above with higher priority.
```

Comment:
This is somewhat confusing. Would it be better to say "The case when TOKEN exists prior in esdtData is handled above with higher priority" or "TOKEN does not exist prior in esdtData because that case is handled above with higher priority"?